CharacterCreation/Class-Specific Documentation/Doctor/Doctor Hippocratic Proceedures.txt

```diff
@@ -0,0 +1,337 @@
+==============================
+Hippocratic Proceedures
+==============================
+
+==Patch Em Up ==
+Adrenaline Reward: 40 Points
+Contagion Reward: 1 Contagion 
+Range:          Touch
+Action:         2 Actions
+    
+    You use your biofoam appllicator to spray the smelly green foam into their
+wounds. The miracle of modern science rapidly reconnects arteries and veins and
+begins to mimic destroyed organs and muscle tissue. It hurts like hell, but it
+will give them a chance to keep it together long enough to get a more permanent
+solution back at base. 
+
+    You must have your Field Applicator drawn in order to use this ability. Your
+patient heals an ammount of health listed on the table below. As your character
+learns more medicine skill, the better they heal. Unless otherwise stated, this
+ability cannot heal a patient above half health.
+
++-----------------+--------------+--------+
+| Med Skill Above | Base Healing |  Dice  |
++-----------------+--------------+--------+
+|       9         |     20       | + 2d10 |
++-----------------+--------------+--------+
+|       19        |     30       | + 3d10 |
++-----------------+--------------+--------+
+|       29        |     35       | + 4d10 |
++-----------------+--------------+--------+
+|       39        |     40       | + 5d10 |
++-----------------+--------------+--------+
+|       49        |     50       | + 6d10 |
++-----------------+--------------+--------+
+
+==Defib ==
+Adrenaline Reward:  80 Points
+Contagion Reward: 2 Contagion
+Range: Touch
+Action: 2 Actions
+
+    You adjust the dials on your biofoam injector to compensate for loss of
+vital fluids and to administer the correct frequency and wattage of tachycardia
+interuptor shock. This is gonna hurt like hell, but they'll live to complain 
+about it at least. Who knows? Maybe they'll even say "Thank you!" this time?
+
+    You must have your Field Applicator drawn in order to use this ability. Only 
+usable on an organic patient that is bleeding out. The person is no longer 
```

Comment:
"Biological" patient would be more accurate here; we leave creative space for non-carbon (and non-synthetic) based life forms.

Reply:
agreed.